scintilla/lexers/LexUser.cxx

```diff
@@ -635,13 +635,13 @@ static inline void SubGroup(const char * s, vvstring & vec, bool group=false)
     for (unsigned int j=0; j<length+1; ++j)
         temp[j] = 0;
 
-    if (s[0] == '(' && s[1]  == '(')

```

Comment:
LexUser is not core scintilla code, but an addon for the UDL from npp , 
see #162 and there https://sourceforge.net/u/nn-n/scintilla/ci/npp_styles/

+#define SCLEX_SEARCHRESULT 150
+#define SCLEX_OBJC 151
+#define SCLEX_USER 152

are additional lexers introduced by npp.